### ISSUE TITLE
CANU failed with exit code 1.  (rc=256)

### ISSUE THREAD

**Issue by bifidoman** [marbl/canu#763]
Hi

CANU 1.6 work very well on my PC, until this horrible day in January....
I got an error every time
- with new input files
- with files that worked fine until yesterday
- with new downloaded Canu (for OSX Mac)
- restart

my command:
```
./canu-1.6/darwin-amd64/bin/canu -p 675SK2_fil -d 675SK2_fil genomesize=5.0m -pacbio-corrected ./vmoutput/675SK2_fil.fastq.txt
```
output (final lines):

```
-- Finished on Tue Jan 23 14:49:50 2018 (lickety-split) with 100.669 GB free disk space
----------------------------------------
ERROR:
ERROR:  Failed with exit code 1.  (rc=256)
ERROR:

ABORT:
ABORT:   Reason: image not found
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
ABORT:   failed to align contigs.
ABORT:
```

Any clues. I fear something is messed up in the BASH, but I don't know what

Thanks

Marc

**Comment by bifidoman** [comment]
Hi

thanks for our support. I rebuild Canu and it was getting better; the "image not found error" was gone, but there was still an error:

ERROR:
ERROR:  Failed with exit code 1.  (rc=256)
ERROR:

ABORT:
ABORT: Canu 1.6
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
ABORT:   failed to align contigs.
ABORT:


The I realised I was always using the same command to test. I changed the target directory, and it worked! (well, at least it is running now, waiting to finnish). I thought Canu would always overwrite existing directories?
